Content.Client/HUD/GameHud.cs

```diff
@@ -606,18 +609,22 @@ public TopButton(Texture texture, BoundKeyFunction function, IInputManager input
                 );
 
                 ToggleMode = true;
+
+                configManager = IoCManager.Resolve<IConfigurationManager>();
             }
 
             protected override void EnteredTree()
             {
                 _inputManager.OnKeyBindingAdded += OnKeyBindingChanged;
                 _inputManager.OnKeyBindingRemoved += OnKeyBindingChanged;
+                configManager.OnValueChanged(CVars.DisplayUSQWERTYHotkeys, x => OnKeyBindingChanged());
```

Comment:
Again, you need to pass the `OnKeyBindingChanged` function directly instead of making a lambda for the (un)subscribe to work correctly.

Also again, subscribe to something on `IInputManager` instead of to the config value.